app/Jobs/DatabaseBackupJob.php

```
@@ -682,6 +681,159 @@ private function getFullImageName(): string
+        if ($config === null) {
+            throw new \Exception('No valid pgBackRest repository configured. Please configure at least one repository (local or S3).');
+        }
+
+        $configBase64 = base64_encode($config);
+        $configPath = PgBackrestService::CONFIG_PATH;
+        $containerName = escapeshellarg($this->container_name);
+
+        instant_remote_process([
+            "docker exec {$containerName} sh -c 'echo {$configBase64} | base64 -d > {$configPath}/pgbackrest.conf'",
+        ], $this->server, true, false, 60, disableMultiplexing: true);
+    }
```

PHPMD (2.15.0)

The method update_pgbackrest_config is not named in camelCase. (undefined)

(CamelCaseMethodName)